src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CommandTable xmlns="http://schemas.microsoft.com/VisualStudio/2005-10-18/CommandTable" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <Extern href="stdidcmd.h"/>
+  <Extern href="vsshlids.h"/>
+
+  <Commands package="guidIssueVizPackage">
+
+    <Groups>
+      <Group guid="guidIssueVizPackageCmdSet" id="SonarLintMenuGroup" priority="0x0100">
+        <Parent guid="guidSHLMainMenu" id="IDM_VS_MENU_TOOLS" />
```

Comment:
The normal place to put "view tool window" commands is on the Tools, Other Window menu.

Reply:
I moved it there. I had to take it out of the "SonarLintMenuGroup" though, since "Other Windows" doesn't seem to allow adding groups.

Reply:
`IDG_VS_WNDO_OTRWNDWS1` is the location used when  you create a new VS template and use the "Async Tool  Window" item template,

The nesting of menu items is a bit weird. If I remember correctly, Groups can contain Menus, and Menus can contain Groups, and Buttons and placed in Groups.

Reply:
I couldn't get the button to show when it's in a group. Based on github everyone is placing the command directly, but it could be because it's the default template as you said. I can place it in a group when a group is under View menu, but not under View -> Other Windows.

Reply:
That's probably because you were trying add a group to a group.
`IDG_VS_WNDO_OTRWNDWS1` is a group - in this case the prefix `IDG-` means "ID of a Group".
You should have been able to place your group on the parent `IDM_VS_MNUCTRL_OTHRWNDS` (where `IDM_` means "ID of a Menu").
Install the [Command Explorer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.CommandExplorer) extension from MadsK. You can use it to see how menus and groups are organised in VS.

Reply:
Ah! thanks! I'll take a look at that and fix accordingly.

Reply:
I don't think there's anything fix - the VSCT is fine as it is.
My comment was just for info to explain why you couldn't add a group to a group.
If you were to add your own group to the menu then the command would appear at a different location, which isn't what we want in this case.